authorizer.py

```diff
@@ -81,19 +81,21 @@ def HasStudentAccess(self):
   def _VerifyServingSession(self, institution, session):
```

Comment:
I think we should remove _VerifyServingSession.  I think the authorizer should report if the user has student access. We could write something else to report which sessions are serving and at what level "preregistration", "preferences", "schedule", or "post registration". Then each page can determine if it can be shown at current serving level.